lib/game_io/test_io.rb

```diff
@@ -24,28 +14,30 @@ def get_input(input=nil)
         when 4
             "4"
         else
-            array = (0..4).to_a
+            array = (0..9).to_a
             array.push("H")
```

Comment:
I did not understand why `"H"` was used (I wasn't sure if it was significant) but then I saw that you have a `PlayerType` module and H is for Human.